docs/source/usage/benchmarks.rst

```diff
@@ -3,6 +3,28 @@
 Benchmarks
 ==========
 
+Parallel benchmark 8
+--------------------
+
+Build based on the helper functions in the :ref:`benchmark utilities <utilities-benchmark>`_, this benchmark executes a simple parallel read-write test.
+
+In particular, this test case writes and reads a 4D array of type ..., sliced 1D along the ... dimension.
+
+.. code-block:: cpp
+
+   openPMD::Extent total{
+       100 * scale_up, // slices along or this...?
```

Comment:
let's place the `// sliced along this axis over MPI ranks` comment on the right axis